opensearch-k8s-operator.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: opensearch-k8s-operator 
+  version: 2.6.1
+  epoch: 0
+  description: OpenSearch Kubernetes Operator
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/opensearch-project/opensearch-k8s-operator 
+      tag: v${{package.version}}
+      expected-commit: b2417013b570cc2dec3acd8204c15f73ba887efa 
+
+  - uses: go/build
+    with:
+      packages: ./opensearch-operator
+      output: manager
+      ldflags: -s -w
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: opensearch-project/opensearch-k8s-operator 
+    strip-prefix: v
+
+test:
+  pipeline:
+    # AUTOGENERATED
+    - runs: |
+        manager 
```

Comment:
What tests, if any, should this PR use? There doesn't seem to be a simple `--version` option. See the `main.go` pf `opensearch-k8s-operator` [here](https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/opensearch-operator/main.go).

Reply:
Okay, I used `--help`.